support/util/util.go

```diff
@@ -655,3 +655,8 @@ func GetControlPlaneOperatorImageLabels(ctx context.Context, hc *hyperv1.HostedC
 
 	return ImageLabels(controlPlaneOperatorImageMetadata), nil
 }
+
+// EnableIfCustomKubeconfig returns true if the hosted control plane has a custom kubeconfig defined
+func EnableIfCustomKubeconfig(hcp *hyperv1.HostedControlPlane) bool {
```

Comment:
nit: the name of the function should reflect what it does. In this case this function is not enabling anything, all it's doing is determining if a KubeAPIServerDNSName is specified.